src/manifest.json

```diff
@@ -31,7 +29,7 @@
     "name": "bumi"
   },
 
-  "__firefox__applications": {
+  "__firefox__browser_specific_settings": {
```

Comment:
It was renamed: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings